### PR TITLE
Sor10874/readmes/36 40

### PR DESCRIPTION
**Pull request by sorenoid**
hi, (famous last words) these were all fairly straightforward, except the Map image layer table example. I also had to remove some API calls from the tags which left  "Map load status" tags with only one tag ("map", smh)

this PR covers 
Map image layer (android sample ':arcgis-mapimagelayer-url')
Change sublayer visibility (android sample ':change-sublayer-visibility')
Map image layer table
Map Initial Extent (android sample ':set-initial-map-area',)
and Map load status

### CODE REVIEW

File: java/map-load-status/README.md [hunk 1]
```diff
@@ -1,23 +1,27 @@
...
+![Image of Map load status](map-load-status.png)
 
 ## How to use the sample
-The `LoadStatus` is considered loaded when any of the following are true:
-* The map has a valid spatial reference
-* The map has an an initial viewpoint
-* One of the map's predefined layers has been created.
 
-A listener is set up on the map to handle the `LoadStatusChangedEvent`, and the status text is updated when the status changes.
+Click on the button to reload the ArcGISMap. The load status of the ArcGISMap will be displayed on screen.
```
Comment (Rachael-E) [inline java/map-load-status/README.md]: let's make this just a little more explicit 
```suggestion
Tap on the refresh button in the top right of the screen to reload the ArcGISMap. The load status of the ArcGISMap will be displayed on screen.
```

File: java/set-initial-map-area/README.md [hunk 2]
```diff
@@ -1,24 +1,33 @@
...
+![Image of map initial extent](set-initial-map-area.png)
+
+## Use case
+
+Setting the initial viewpoint is useful when a user wishes to first load the map at a particular area of interest. 
 
 ## How to use the sample
-Simply run the app.
+
+As application is loading, initial view point is set and map view opens at the given location.
```
Comment (Rachael-E) [inline java/set-initial-map-area/README.md]:
```suggestion
As the application is loading, the initial view point is set and the map view opens at the given location.
```

**Comment by sorenoid** [comment]
pinging @hud10837 for 2nd review.

File: java/map-load-status/README.md [hunk 3]
```diff
@@ -1,23 +1,27 @@
...
+
+* The map has a valid spatial reference.
+* The map has an an initial viewpoint.
+* One of the map's predefined layers has been created.
 
 ## Relevant API
+
 * ArcGISMap
-* LoadStatusChangedListener
+* LoadStatusChangedListener/LoadStatusChanged signal
```
Comment (hud10837) [inline java/map-load-status/README.md]: This seems like a strange line?

Comment (sorenoid) [reply java/map-load-status/README.md]: yes.

**Comment by sorenoid** [comment]
@hud10837 if everything looks good, please ~click~ tap the approve button

**Comment by sorenoid** [comment]
@Rachael-E Is it ok to merge this one?

**Comment by Rachael-E** [comment]
go for it!

**Pull request merged**